contributrs-quick-start.rst

```diff
@@ -0,0 +1,976 @@
+
+
+Docker Community Edition
+-----------------------------------
+
+
+1. Installing required packages for Docker and setting up docker repo
+
+.. code-block:: bash
+
+  $ sudo apt-get update
+
+  $ sudo apt-get install \
+      apt-transport-https \
+      ca-certificates \
+      curl \
+      gnupg-agent \
+      software-properties-common
+
+  $ curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+
+  $ sudo add-apt-repository \
+     "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+     $(lsb_release -cs) \
+     stable"
+
+2. Install Docker
+
+.. code-block:: bash
+
+  $ sudo apt-get update
+  $ sudo apt-get install docker-ce docker-ce-cli containerd.io
+
+3. Creating group for docker and adding current user to it.
+
+.. code-block:: bash
+
+  $ sudo groupadd docker
+  $ sudo usermod -aG docker $USER
+
+Note : After adding user to docker group Logout and Login again for group membership re-evaluation.
+
+4. Test Docker installation
+
+.. code-block:: bash
+
+  $ docker run hello-world
+
+
+
+
+Docker Compose
+--------------------------
+
+1. Installing latest version of Docker Compose
+
+.. code-block:: bash
+
+  $ COMPOSE_VERSION="$(curl -s https://api.github.com/repos/docker/compose/releases/latest | grep '"tag_name":'\
+  | cut -d '"' -f 4)"
+
+  $ COMPOSE_URL="https://github.com/docker/compose/releases/download/${COMPOSE_VERSION}/\
+  docker-compose-$(uname -s)-$(uname -m)"
+
+  $ sudo curl -L "${COMPOSE_URL}" -o /usr/local/bin/docker-compose
+
+  $ sudo chmod +x /usr/local/bin/docker-compose
+
+2. Verifying installation
+
+.. code-block:: bash
+
+  $ docker-compose --version
+
+
+
+pyenv and setting up virtual-env
+--------------------------------------------
+
+1. Checking required packages
+
+.. code-block:: bash
+
+  $ sudo apt-get install -y build-essential libssl-dev zlib1g-dev libbz2-dev \
+      libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+      xz-utils tk-dev libffi-dev liblzma-dev python-openssl git
+
+  $ sudo apt install build-essentials python3.6-dev python3.7-dev python3.8-dev python-dev openssl \
+       sqlite sqlite-dev default-libmysqlclient-dev libmysqld-dev postgresql
+
+2. Install pyenv
+
+.. code-block:: bash
+
+  $ curl https://pyenv.run | bash
+
+3. Add the lines suggested at the end of installation  to ~/.bashrc
+
+4. Restart your shell so the path changes take effect and verifying installation
+
+.. code-block:: bash
+
+  $ exec $SHELL
+  $ pyenv --version
+
+5. Checking available version, installing required Python version to pyenv and verifying it
+
+.. code-block:: bash
+
+  $ pyenv install --list
+  $ pyenv install 3.8.5
+  $ pyenv versions
+
+6. Creating new virtual environment named ``airflow-env`` for installed version python
+
+.. code-block:: bash
+
+  $ pyenv virtualenv 3.8.5 airflow-env
+
+7. Entering virtual environment ``airflow-env``
+
+.. code-block:: bash
+
+  $ pyenv activate airflow-env
+
+
```

Comment:
Maybe add a comment here that you will install airflow in the virtual env in the next step. It's not clear why empty env is created here.